ipatests/prci_definitions/nightly_latest_pki.yaml

```diff
@@ -783,6 +783,18 @@ jobs:
         timeout: 3600
         topology: *master_1repl
 
+  pki-fedora/test_ipahealthcheck_TestIpaHealthCheck:
+    requires: [fedora-latest/build]
+    priority: 50
+    job:
+      class: RunPytest
+      args:
+        build_url: '{pki-fedora/build_url}'
```

Comment:
1. Within this file the job definition needs to have `update_packages: True` to make sure to use the nightly build of pki.
2. Does this test really need to be executed for dogtag regression testing?